stripe/src/main/java/com/stripe/android/model/Card.java

```diff
@@ -211,6 +280,64 @@ public Card build() {
     }
 
     /**
+     * Create a Card object from a raw JSON string.
+     *
+     * @param jsonString the JSON string representing the potential Card
+     * @return A Card if one can be made from the JSON, or {@code null} if one cannot be made
+     * or the JSON is invalid.
+     */
+    @Nullable
+    public static Card fromString(String jsonString) {
+        try {
+            JSONObject jsonObject = new JSONObject(jsonString);
+            return fromJson(jsonObject);
+        } catch (JSONException ignored) {
+            return null;
+        }
+    }
+
+    @Nullable
+    public static Card fromJson(JSONObject jsonObject) {
+        if (jsonObject == null || !VALUE_CARD.equals(jsonObject.optString(FIELD_OBJECT))) {
+            return null;
+        }
+
+        Integer expMonth = optInteger(jsonObject, FIELD_EXP_MONTH);
+        Integer expYear = optInteger(jsonObject, FIELD_EXP_YEAR);
+
+        //It's okay for the month to be missing, but not for it to be outside 1-12
+        if (expMonth != null && (expMonth < 1 || expMonth > 12)) {
+            return null;
+        }
+
+        if (expYear != null && expYear < 0) {
+            return null;
+        }
```

Comment:
how did we decide that we only need to check the `expMonth` and `expYear`? should we add more checks?